api/src/main/resources/workbench.yaml

```diff
@@ -111,6 +111,27 @@ paths:
           schema:
             $ref: "#/definitions/UsernameTakenResponse"
 
+  /api/v1/invitationKeyVerification:
```

Comment:
Any particular reason you chose camel-case here instead of hyphen-separated?

Reply:
It was just a personal Choice, i prefer camelCase over hyphen, but if hyphen is the norm i can change that

Reply:
Since this is a URL, I have urged the team to move toward hyphen-separated words. URLs (even API URLs occasionally) tend to wind up in a user's browser address bar. We shouldn't expect users to understand case-sensitivity and, for similar reasons, I also discourage snake-case (foo_bar) because underscores are often hard to spot.